src/lib/component/util/useReqContext.tsx

```diff
@@ -0,0 +1,15 @@
+import { parseStringParam } from "/lib/util/params";
+import { useRouter } from "/lib/component/util";
+
+const useReqContext = () => {
+  const router = useRouter();
+  const params = router.match.params as any;
+  const simpleUrl = !router.location.pathname.includes("~");
+
+  return {
+    namespace: parseStringParam(params.namespace, ""),
+    cluster: parseStringParam(params.cluster, (simpleUrl ? "" : "~")),
```

Comment:
```suggestion
    cluster: parseStringParam(params.cluster, ""),
```

I wonder if we could omit the `simpleUrl` logic 🤔. I theory..
* in OSS the value of `params.cluster` should _always_ be `undefined`
* in the enterprise product `params.cluster` should always at least be `"~"` or a string with one or more characters.

Reply:
Discussed IRL, making this change would break the links. Resolving.